### DISCUSSION TITLE
【BUG】it seems that seed_random is not effective anymore

### DISCUSSION THREAD

**Discussion by diaolulu1** [lllyasviel/Fooocus#806] [Show and tell]
After the support for wildcards was added, it seems that seed_random is not effective anymore. Regardless of whether I check seed_random or not, the seed value keeps increasing based on the image_number. Even if I provide a specific seed value, it still increases according to the image_number. I need to achieve the following functionality: when seed_random is checked, the seed value should not be influenced by the image_number, and it should be a random value each time. When seed_random is not checked, I want to be able to enter a specific seed value to keep it fixed, and only change the wildcard representations of prompt words to generate the desired number of images.

I Fixed the BUG。
[modules.zip](https://github.com/lllyasviel/Fooocus/files/13196558/modules.zip)

**Comment by lllyasviel** [comment]
duplicated with https://github.com/lllyasviel/Fooocus/issues/799
